test/modules/packages/blas/test_blas2.chpl

```diff
@@ -0,0 +1,1690 @@
+use Random;
+use BLAS;
+use blas_helpers;
+
+
+proc main() {
+    // TODO -- band Matrices & packed Matrices
```

Comment:
Can you add another TODO (unless I missed it somewhere) concerning testing for compile-time and execution-time errors?